pkg/server/api/api_test.go

```
@@ -0,0 +1,85 @@
+	"net"
+	"net/http"
+	"testing"
+)
+
+func startTestUnixSocketServer(t *testing.T) (string, <-chan string) {
+	t.Helper()
+
+	tmpDir := t.TempDir()
+	pathCh := make(chan string, 1)
+
+	listener, err := net.Listen("unix", SocketPath(tmpDir))
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)